docs/tickets/301/plan.md

```
@@ -0,0 +1,244 @@
+**Config** (`libs/third-party-user/src/config.ts`):
+```typescript
+export const prismaSchemas = path.join(__dirname, "../prisma");
+```
+
+**Package**: `@hmcts/third-party-user`
+
+### 2.2 New pages in `libs/system-admin-pages`
+
+Each page follows the existing pattern: `index.ts` (controller), `en.ts`, `cy.ts`, `index.njk`, `index.test.ts`.
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)